### ISSUE TITLE
Separate Section 15 (OGL) for each entry

### ISSUE THREAD

**Issue by Sanglorian** [5srd/5srd.github.io#10]
I recommend storing the Section 15 information for each entry in the entry itself, and then generating a copy of the Open Gaming License for each entry, populated with that entry's Section 15.

I think this has two main benefits:

1. It makes it easier to reuse 5srd content elsewhere, because the Section 15 won't be unmanagably long. 
2. It makes it easier to figure out where each entry is from, and what sources it's based on. 

What do you think?

**Comment by drumanagh** [comment]
I think the actual mechanics of this would be relatively straightforward, though for anything third-party there'd be a lot of duplicated content (e.g. monsters A, B, and C with the same section 15 entries citing half-a-dozen books).

The main problem at the moment is stuff like the replacement fluff text pulling from the Pathfinder books would need a huge Section 15 on every monster page, and one that's distinct about being for the fluff text but not for the stats.

Also, for sourcing purposes every content entry should now have a "Source" marker at the upper right - please open an issue or whatever if you spot one that doesn't.

**Comment by Sanglorian** [comment]
Hmm, good points.

What about a Section 15 entry for every source instead? Then on the imp page, they can click on "5E SRD" and see the Section 15 for that source, or click on Pathfinder Bestiary and see its Section 15?

**Comment by drumanagh** [comment]
I don't think that would really help. The OGL is for legal acknowledgment, not sourcing, and doing that wouldn't remove the requirement to still have a notice on each individual entry or on the site as a whole.

**Comment by Sanglorian** [comment]
But it would allow re-users to take only the Section 15 entries that they need for the entries that they're using. 

I think Grand OGL Wiki did something like it:

{text of OGL}
{section 15 of source}

and then a master entry:

{text of OGL}
{all section 15s}
